src/components/@shared/FormInput/InputElement/FilesInput/index.tsx

```diff
@@ -32,6 +32,13 @@ export default function FilesInput(props: InputProps): ReactElement {
           'Google Drive is not a supported hosting service. Please use an alternative.'
         )
       }
+      // Check if provider is a valid provider
+      const isValid = await ProviderInstance.isValidProvider(providerUrl)
+
+      if (!isValid)
+        throw Error(
+          '✗ Provider cannot be reached, please check status.oceanprotocol.com and try again later.'
```

Comment:
i think here we can add a reference to custom or self hosted provider as well, maybe adding also something like this at the end of the error message: `if you have your own hosted provider please check the status of the service`

Reply:
I'm not sure if that's necessary, as they should be stoped at the previous step if they have their own provider that doesn't work. If they enter there own provider then they have to click validate on the publish form, which shows them a separate error message:

![image](https://user-images.githubusercontent.com/20739535/213215185-44807b02-982e-4ed0-8c62-a0cef178c0a5.png)